src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -167,6 +167,24 @@ public static String getDefaultAlias(String sourceName) {
         return sourceName.substring(startIndex, endIndex).toLowerCase(Locale.ENGLISH);
     }
 
+    /**
+     * Returns the default name for the source name of a check, which is the
+     * source name with any dotted prefix or "Check" suffix
+     * romoved.
+     *
+     * @param sourceName the source name of the check (generally the class
+     *        name)
+     * @return the default name for the given check
+     */
+    public static String getDefaultName(String sourceName) {
+        int endIndex = sourceName.length();
```

Comment:
This code is very similar to getDefaultAlias
Let's reuse code as much as possible